### DISCUSSION TITLE
Implementable status for KubeEdge Device Management

### DISCUSSION THREAD

**Discussion by smoms** [kubeedge/kubeedge#4331] [General]
@rohitsardesai83 I am interested in Device Management https://github.com/kubeedge/kubeedge/blob/master/docs/proposals/device-crd.md.
I see that the status of the requirements are _implementable_, how do I know then the current status?

thanks.

**Comment by vincentgoat** [comment]
cc @RyanZhaoXB

**Comment by RyanZhaoXB** [comment]
hello. KubeEdge Device Management has been implemented and applicated for a lot of scenarios. and now we are designing the next stage of device management interface (DMI). you can get the information in this issue https://github.com/kubeedge/kubeedge/issues/3760. please feel free to contact me if you have any problems or suggestions about KubeEdge device management.

**Reply by smoms** [reply]
Ok many thanks. I am just wondering how to interpreter the status **implementable** (https://github.com/kubeedge/kubeedge/blob/master/docs/proposals/device-crd.md). Is something that we can assume to be already implemented?
